index.html

```diff
@@ -0,0 +1,571 @@
+directive @core(feature: String)
+```
+
+### `feature: `[core__FeatureUrl](#core__featureurl)
+
+A [feature URL](#core__FeatureUrl) specifying the directive and associated schema elements. When viewed, the URL **should** provide the content of the appropriate version of the specification in some human-readable form. In short, a human reader should be able to click the link and go to the docs for the version in use. There are specific requirements on the format of the URL, but it is not required that the *content* be machine-readable in any particular way.
+
+<figure>
+  <figcaption>
+    <a name=anatomy-of-a-feature-url href=#anatomy-of-a-feature-url class=a-header>Basic anatomy of a feature URL</a>
+  </figcaption>  
+  <code class=anatomy>
+    <span class=pink style='--depth: 2'>http://spec.example.com/a/b/c/<span>exampleSpec<aside>name</aside></span><aside>
+      identity
+    </aside></span>/<span style='--depth: 2' class=green>v1.0<aside>version</aside></span>
+  </code>
+</figure>
+
+Feature URLs contain information about the spec's [prefix](#prefixing) and [version](#versioning), and should be generated and processed in accordance with the [requirements on the structure of feature URLs](#feature-urls).
+
+### `as: String`
+
+Change the [names](#prefixing) of directives and schema elements from this specification. The specified string **must** be a valid GraphQL identifier and **must not** contain the namespace separator (two underscores, `__`).
+
+When `as:` is provided, processors **must** replace the default name prefix on the names of all [prefixed schema elements](#elements-which-should-be-prefixed) with the specified name.
+
+```graphql
+#::[Using `@core(feature:, as:)` to use a feature with a custom name]
+schema
+  @core(feature: "https://spec.example.com/core/v1.0")
+  @core(feature: "https://spec.example.com/example/v1.0", as: "eg")
+{
+  query: Query
+}
+
+type User {
+  # Specifying `as: "eg"` transforms @example into @eg
+  name: String @eg(data: ITEM)
+}
+
+# Additional specified schema elements must have their prefixes set
+# to the new name.
+#
+# This data enum was specified as `example__Data`, but will be renamed
+# as `eg__Data`:
+enum eg__Data {
+  ITEM
+}
+
+# Name transformation must also be applied to definitions pulled in from
+# specifications.
+directive @eg(data: eg__Data) on FIELD
+
+# (...other definitions omitted...)
+```
+
+## `@core__export`
+
+Control the export of individual schema elements.
+
+```graphql
+#::[The `@core__export` directive]
+directive @core__export(isExport: Boolean = true)
+  repeatable on
+  | SCALAR
+  | OBJECT
+  | FIELD_DEFINITION
+  | ARGUMENT_DEFINITION
+  | INTERFACE
+  | UNION
+  | ENUM
+  | ENUM_VALUE
+  | INPUT_OBJECT
+  | INPUT_FIELD_DEFINITION
+```
+
+`@core__export` can occur at any type system location. Elements with `@core__export` will always be included in the export schema. Elements with `@core__export(isExport: false)` will always be excluded from the export schema.
```

Comment:
I'd suggest linking to one of the two other places that define how exporting works to explain what happens to elements without this directive.